spec/ffprober/parser_spec.rb

```diff
@@ -26,6 +26,24 @@ def shared_specs
     end
   end
 
+    describe 'subtitle_streams' do
+    it 'should determine the correct duration_ts of the first subtitle stream' do
+      expect(ffprobe.subtitle_streams.first.duration_ts).to eq(98)
+    end
+
+    it 'should determine the correct bitrate of the first subtitle streams' do
+      expect(ffprobe.subtitle_streams.first.bit_rate).to eq('5690')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.